src/components/share/ShareDirective.js

```diff
@@ -47,6 +47,19 @@ goog.require('ga_permalink');
         }
         // Use clipboard API to copy URL in OS clipboard
         element.on('click', function() {
+
+            // the copy link for the iframe-preview is refreshed
+            // according to the preview changes
+            var previewFrame;
+            [previewFrame] = angular.element(document).find('iframe');
```

Comment:
Using an id seems inevitable. Use the id of the modal `#gaEmbedModal iframe`for now. I have the feeling that all this piece of code should be elsewhere. and not in this directive.